lib/DDG/Spice/WordMap.pm

```diff
@@ -12,38 +12,38 @@ category 	"reference";
 topics 		"words_and_games", "everyday";
 code_url 	"https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/WordMap.pm";
 icon_url 	"/i/www.twinword.com.ico";
-attribution web 	=> ['http://twinword.com','Twinword Inc.'],
-            twitter => ['http://twitter.com/levelpump', 'Twinword Inc.'],
+attribution web 	=> ['https://www.twinword.com','Twinword Inc.'],
+            twitter => ['https://twitter.com/twinwordinc', 'Twinword Inc.'],
             email 	=> ['feedback@twinword.com','Twinword Inc.'];
 
-spice to => 'http://duckduckgo.twinword.com/api/v2/context/user/duckduckgo/?entry=$1&api_key={{ENV{DDG_SPICE_WORDMAP_APIKEY}}}';
+spice to => 'https://www.twinword.com/api/v2/context/user/duckduckgo/?entry=$1&api_key={{ENV{DDG_SPICE_WORDMAP_APIKEY}}}';
```

Comment:
I know your point though, Twinword's API policy is forcing to use SSL with all partners. To my understanding DDG has good cache system. so, I'd like to stick with https. Is it possible?